thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/AutotuneConfigDTO.java

```diff
@@ -38,6 +38,12 @@ public void setAlertFilter(AlertFilter alertFilter) {
     this.alertFilter = alertFilter;
   }
 
+  public void setAlertFilter(BaseAlertFilter baseAlertFilter) {
```

Comment:
The convention for setXXX is to set the value of XXX field. Using setAlertFilter to set both alert filter and tuning properties is not a good idea. I would suggest to name it initAlertFilter or so.

Reply:
fixed.